pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/JdbiI18nTranslation.sql.stg

```diff
@@ -0,0 +1,17 @@
+group JdbiI18nTranslation;
+
+getI18nTranslations() ::= <<
+select
+        t.i18n_translation_id,
+        t.i18n_translation_doc,
+        t.study_id,
+        t.language_code_id,
+        t.created_at,
+        t.updated_at
+from
+    i18n_translation t
```

Comment:
I believe i18n_translation is a new table.
Not seeing the Liquibase script/schema change to create this table !

Reply:
ok. I see it as a separate PR https://github.com/broadinstitute/ddp-study-server/pull/1048/files. 
May be that schema change files should be added to this PR

Reply:
Yes, I created Liquibase script in a separate PR. May be I should copy the script to this PR and close PR 1048..

Reply:
@ssettipalli I added the Liquibase script creating table `i18n_translation` to this PR. And PR https://github.com/broadinstitute/ddp-study-server/pull/1048 is closed.